README.md

```diff
@@ -396,6 +398,9 @@ British Machine Vision Conf. (BMVC), London, 2017.
 - Delbruck, T., Villanueva, V., Longinotti, L.,  
 *[Integration of dynamic vision sensor with inertial measurement unit for electronically stabilized event-based vision](http://doi.org/10.1109/ISCAS.2014.6865714)*,  
 IEEE Int. Symp. Circuits and Systems (ISCAS) 2014, 2636-2639. [YouTube](https://youtu.be/Tzy4WF6Qp-Y)
+- Vasco, V., Glover, A., Mueggler, E., Scaramuzza, D., Natale, L., and Bartolozzi, C.  
+*[Independent Motion Detection with Event-driven Cameras](https://arxiv.org/abs/1706.08713v2)*,  
+IEEE/RAS Int. Conf. on Advanced Robotics (ICAR) 2017, Hong Kong, China.
 
```

Comment:
This is a duplicate. Elias suggested section `Localization and Ego-Motion Estimation`. Should it be on both sections or just in one?